ui/pages/DataManagement/DataManagement.jsx

```diff
@@ -10,6 +10,7 @@ import ElasticsearchStatus from './components/ElasticsearchStatus'
 import RnaSeq from './components/RnaSeq'
 import SampleQc from './components/SampleQc'
 import Users from './components/Users'
+import PhenotypePri from './components/PhenotypePri'
```

Comment:
as above, `Pri` sounds like an acronym and not an abbreviation so its confusing. `PhentoypePriority` is not that long of a class name